uber/templates/regform.html

```diff
@@ -357,8 +357,9 @@
 <div class="form-group staffing">
     <label for="staffing" class="col-sm-3 optional-field control-label">Want to Volunteer?</label>
     <div class="checkbox col-sm-6">
-        <label for="staffing">
-        {{ macros.checkbox(attendee, 'staffing') }}
+        {% set staffing_readonly = not admin_area and attendee.shifts %}
+        <label for="staffing" {% if staffing_readonly %}title="Please {{ 'contact your department head to drop shifts' if c.AT_THE_CON else 'unassign yourself from shifts' }} before changing your volunteer status."{% endif %}>
```

Comment:
I put the `title` attribute on the `label` instead of on the `checkbox`, so the user gets the hover text no matter where they're hovering.